### DISCUSSION TITLE
Import postman collection

### DISCUSSION THREAD

**Discussion by nishuzumi** [darrenburns/posting#71] [Ideas]
Is there any way to import postman collection?

**Comment by AssassinPrince** [comment]
Hey There
As for now this project only imports `OpenAPI` yaml formats, You need to use a convertor.
There are plenty to choose from, I dont want to suggest you one cause I'm just starting to use this project and wont know which one is better or worse
ed1. And also importing OpenAPI feature is in `highly experimental` stage as told in README.

Postman suggests some convertors but I couldnt get any complete conversion out of 'em, something was always missing.
